spec/tests/QE/test_no_conflicts.md

```diff
@@ -0,0 +1,60 @@
+# Test Cases
+
+## #1 test_sg_cbl_updates_concurrently_with_push_pull
+
+### Description
+Test concurrent updates to the same document in both Sync Gateway and Couchbase Lite, and verify that conflicts are resolved and changes are replicated correctly.
+
+### Steps
+1. Reset SG and load `posts` dataset
+2. Reset local database and load `posts` dataset
+3. Pull replication (continuous) to CBL
+4. Wait until the replicator is idle
+5. Create docs in SG
+6. Update docs in SGW and CBL
+7. Start Push Pull replication between SGW and CBL (continuous)
```

Comment:
You are going to start another continuous replication while the previous continuous replication is running?